docs/reference/release-notes/6.4.asciidoc

```diff
@@ -55,8 +55,12 @@ coming[6.4.1]
 //[float]
 //=== Enhancements
 
-//[float]
-//=== Bug Fixes
+[float]
+=== Bug Fixes
+
+Rollover::
+{es} can once again start if any shards on the node have been
+<<indices-rollover-index, rolled over>>. {pull}33394[#33394]
```

Comment:
I think this needs to wait to be added until 6.4.1 release time?

Reply:
We're already generated 6.4.1 release notes with a "coming in 6.4.1" note. I think it is ok to add this to that page.